components/google-cloud/OWNERS

```diff
@@ -0,0 +1,20 @@
+# Google Cloud components - MB SDK
+approvers:
+  - Ark-kun
+  - Bobgy
+  - chensun
+  - IronPan
+  - neuromage
+  - paveldournov
+  - sasha-gitg
+  - sinachavoshi
+reviewers:
+  - Ark-kun
+  - Bobgy
+  - chensun 
+  - sasha-gitg
+  - sinachavoshi
+
+# TODO() Add the owners specified below to `CODEOWNERS` file in root so that they are used for PRs.
+# /components/google_cloud_components/mb_sdk_component/* sasha-gitg sinachavoshi
+# /components/containers/uCAIP_MB_SDK/* sasha-gitg sinachavoshi
```

Comment:
I think you want to keep this restricted to the MB team for now

Reply:
acknowledged, I wanted to make sure it has parity with /pipeline/components/google-cloud .. I will do a follow up PR to clean up the owners (tracked under b/184538644)